interop-testing/src/main/java/io/grpc/testing/integration/XdsTestClient.java

```diff
@@ -260,6 +260,7 @@ private static RpcType parseRpc(String rpc) {
     }
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
This suppressed deprecation for many things. We do want deprecation warnings. It is better to put the annotation on an assignment:
```java
@SuppressWarnings("deprecation")
BindableService oldReflectionService = ProtoReflectionService.newInstance();
```

Ditto for the other files.